src/backend/tests/c-lisp/conditional.sexp

```diff
@@ -0,0 +1,12 @@
+(c-lisp
+    (define ((print int) (n int)))
+
+    (define ((main int))
+        (if #t
```

Comment:
We can interchangeable use [xyz] and (xyz). So we can use syntax that makes stuff explicit in sexp, but doesn't really matter. Just a syntactic sugar.

Example

(if (xyz)
    [statements1]
   [statements2])

Actually we can follow that square backets are used for block statements. It's just a convention.